Description: Extend array dependency range for GHC 7.4.1
Author: Joachim Breitner <nomeata@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- haskell-representable-functors-2.1.orig/representable-functors.cabal
+++ haskell-representable-functors-2.1/representable-functors.cabal
@@ -1,6 +1,6 @@
 name:          representable-functors
 category:      Monads, Functors, Data Structures
-version:       2.1
+version:       2.1.0.0.1
 license:       BSD3
 cabal-version: >= 1.6
 license-file:  LICENSE
@@ -30,7 +30,7 @@ library
     UndecidableInstances

   build-depends:
-    array                >= 0.3.0.2 && < 0.4,
+    array                >= 0.3.0.2 && < 0.5,
     base                 >= 4       && < 5,
     comonad              >= 1.1.1.1 && < 1.2,
     containers           >= 0.3     && < 0.5,
